package.json

```diff
@@ -98,6 +98,7 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
     "mocha": "^2.2.5",
+    "nodemon": "^1.9.1",
```

Comment:
@oliviertassinari So maybe adding to `devDependencies` is a good idea given the usage here (as a library specific dev tool rather than a CLI tool for the user's machine)